src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is implemented to handle cases where some properties might be missing, such as `age` for `olya` and `partnerName` for `alex`. This will prevent potential errors or warnings when rendering these components.